lib/kwtool/accounts/schemas/user.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Kwtool.Accounts.Schemas.User do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Argon2
```

Comment:
Maybe we can omit this cause we can just call `Argon2.add_hash` No need alias in this case 🤔